src/Icon/icon.css

```diff
@@ -10,7 +10,31 @@
     white-space: nowrap;
     word-wrap: normal;
     direction: ltr;
-    font-feature-settings: 'liga';
-    -webkit-font-feature-settings: 'liga';
+    font-feature-settings: "liga";
+    -webkit-font-feature-settings: "liga";
     -webkit-font-smoothing: antialiased;
 }
+
+.d2ui-align-icon {
```

Comment:
Tab and Button needed the same icon/label alignment, so I've created a helper class in the Icon's CSS instead of duplicating code in the button and Tab

Reply:
👍